mautic/mautic
# Move batch rebuild button to the batch dropdown

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n


[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR simply moves the form cache rebuild button from the toolbar to be with the rest of the batch actions in the check all dropdown.

#### Steps to test this PR:
1. Go to forms, check some, click the batch dropdown around the checkall inbox, use the new rebuild cache button
